client/src/components/CustomSearchBar.js

```diff
@@ -9,7 +9,7 @@ import LoginFormStyler from '../utils/LoginFormStyler';
 import axios from 'axios';
 import {useGlobalState} from '../context/GlobalState';
 import {PICTURES_API} from '../consts/awsConsts';
-import ReviewCard from './ReviewCard';
+import ReviewCard from './ReviewCard/ReviewCard';
```

Comment:
nit: update the ReviewCard folder name to Review so that the file and the folder do not have the same name